lizmap/modules/lizmap/lib/Project/UserFilesFinder.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @author    3Liz
+ * @copyright 2024 3liz.com
+ *
+ * @see      3liz.com
+ *
+ * @license   MIT
+ */
+
+namespace Lizmap\Project;
+
+class UserFilesFinder
```

Comment:
I would call it `projectFilesFinder` instead, even if in the future, list of files may depend on the authenticated user.